fili-core/src/main/java/com/yahoo/bard/webservice/table/PhysicalTable.java

```diff
@@ -38,7 +40,7 @@
     private Map<Column, Set<Interval>> workingIntervals;
     private final Object mutex = new Object();
     private final Map<String, String> logicalToPhysicalColumnNames;
-    private final Map<String, String> physicalToLogicalColumnNames;
+    private final Map<String, List<String>> physicalToLogicalColumnNames;
```

Comment:
It's really more about avoiding overspecification. The places we use it only needs a `Collection` (we're iterating over every element). And at least in the current implementation we don't seem overly concerned with uniqueness. Tough having the same dimension twice could lead to weirdness if `addNewDimensionColumn` isn't idempotent.